README.md

```diff
@@ -146,6 +146,7 @@ kube-capacity --pods --containers --util --output yaml
       --kubeconfig string         kubeconfig file to use for Kubernetes config
   -n, --namespace string          only include pods from this namespace
       --namespace-labels string   labels to filter namespaces with
+      --no-taint                  exclude nodes with taints
```

Comment:
I don't think this PR has merged yet (https://github.com/robscott/kube-capacity/pull/98). Mind leaving this change out until I can figure out how to handle that PR and/or maybe just rebase it myself?

Reply:
the --no-taint flag was added [in this commit](https://github.com/robscott/kube-capacity/commit/a68c7d4871db91edf7f7dbd69ee273cbbc8181af). 

Flag is initialized [here](https://github.com/robscott/kube-capacity/blob/main/pkg/cmd/root.go#L75). You can see it on main currently with `-h`.

Apologies I shouldn't have linked issue #91 because as you pointed out that has an open PR. #98 which is related to excluding specific taints.

Reply:
🤦 you're completely right, thanks!